DigitalOffice/Services/ProjectService/Validators/EditProjectValidator.cs

```diff
@@ -0,0 +1,37 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.ProjectService.Models;
+using System;
+
+namespace LT.DigitalOffice.ProjectService.Validators
+{
+    public class EditProjectValidator : AbstractValidator<EditProjectRequest>
+    {
+        public EditProjectValidator()
+        {
+            RuleFor(project => project.Id)
+                .Must(NotGuidEmpty)
+                .WithMessage("Request must have a project Id");
+
+            RuleFor(project => project.Name)
+                    .NotEmpty()
+                    .WithMessage("Project must have a name.")
+                    .MaximumLength(80)
+                    .WithMessage("Project name too long.")
+                    .Matches("^[A-Z 0-9][A-Z a-z 0-9]+$")
+                    .WithMessage("Incorrect project name.");
```

Comment:
remove, in project name can be any characters.

Reply:
alright then, I just looked at the new project and since they're kind of the same...